<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manage_shared_with</class>
 <widget class="QDialog" name="manage_shared_with">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>11</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_shared_with">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>71</y>
     <width>301</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>361</y>
     <width>301</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Overall Quota:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_views_2"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_update_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>update views</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Usernames | agreed quota | current views :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
